src/components/flowchart-wrapper/flowchart-wrapper.js

```diff
@@ -78,6 +83,51 @@ export const FlowChartWrapper = ({
     matchedFocusedNode,
   } = findMatchedPath(pathname, search);
 
+  /**
+   * Sets the query params from local storage based on NodeType, tag, expandAllPipelines and active pipeline.
+   * @param {string} activePipeline - The active pipeline.
+   */
+  const setParamsFromLocalStorage = (activePipeline) => {
```

Comment:
does this function only used on initial load ? like when the user is not modifying filters ? Based on the not conditions like - `if (!searchParams.has(params.tags))`, I think that is the case. If so, can we add that in the comment on line 87, just to be clear ?

Reply:
@ravi-kumar-pilla Yes this function used on initial load and when there will be any changes in active pipeline. same updated in function comment.